buildlib/tools/coverity.sh

```diff
@@ -1,44 +1,43 @@
 #!/bin/bash -eExl
-reldir=$(realpath $(dirname $0))
 
-source ${reldir}/common.sh
+realdir=$(realpath $(dirname $0))
+source ${realdir}/common.sh
+source ${realdir}/../az-helpers.sh
+
+COV_MODULE="tools/cov"
+
 #
 # Run Coverity and report errors
 # The argument is a UCX build type: devel or release
 #
-
 modules_for_coverity() {
 	res=0
-	module_load $COV_MODULE
+	az_module_load $COV_MODULE
 	res=$(($res+$?))
-	module_load $CUDA_MODULE
+	az_module_load $CUDA_MODULE
 	res=$(($res+$?))
-	module_load $GDRCOPY_MODULE
+	az_module_load $GDRCOPY_MODULE
 	res=$(($res+$?))
-	module_load $JDK_MODULE
+	az_module_load $JDK_MODULE
 	res=$(($res+$?))
-	module_load $MVN_MODULE
+	az_module_load $MVN_MODULE
 	res=$(($res+$?))
-	module_load $XPMEM_MODULE
+	az_module_load $XPMEM_MODULE
 	res=$(($res+$?))
 	return $res
 }
 
 run_coverity() {
 
+	az_init_modules
 	modules_for_coverity
 
-	if [ $res -gt 0 ]; then
-		azure_log_warning "All modules cannot load"
-	fi
-
 	ucx_build_type=$1
 
-	xpmem_root=$(module show dev/xpmem-latest 2>&1 | awk '/CPATH/ {print $3}' | sed -e 's,/include,,')
+	xpmem_root=$(module show $XPMEM_MODULE 2>&1 | awk '/CPATH/ {print $3}' | sed -e 's,/include,,')
```

Comment:
as we discussed before - need to fix src/uct/sm/mm/xpmem/configure.m4 so it will search xpmem by only using AC_CHECK_HEADER and not by "find" command. this way, loading xpmem module will set "CPATH" and "INCLUDE" env vars, which will be used by AC_HEAD_HEADER to get the right path.

Reply:
it 's ok , but let do it in other PR , currently it is not so priority for me

Reply:
ok